src/construction.cpp

```diff
@@ -313,14 +337,19 @@ void construction_menu()
 
             if( !constructs.empty() ) {
                 std::string current_desc = constructs[select];
+                mvwprintz( w_con, w_height - 5, ( w_list_width + w_list_x0 + 2 ), c_white,
```

Comment:
It seems that I can't comment on lines you didn't touch so I'll put it here:
line 310 / 334 should be updated to this:
```int available_buffer_height = w_height - 7 - 3;``` to solve issue 1 from my latest comment

Extra points if you will replace magic number ```7``` by constant/variable with some good name (which will describe its purpose).